README.md

```diff
@@ -1108,7 +1108,7 @@ For more, see [pygrep-hooks](https://github.com/pre-commit/pygrep-hooks) on GitH
 | PGH003 | blanket-type-ignore | Use specific rule codes when ignoring type issues |  |
 | PGH004 | blanket-noqa | Use specific rule codes when using `noqa` |  |
 
-### Pylint (PLC, PLE, PLR, PLW)
```

Comment:
I feel like this was changed intentionally at some point, to spell out all of the rules, but maybe that was actually for Pycodestyle, and Pylint ended up this way as a result of that convention?

Reply:
I don't know the history behind this  line. But since 9dc66b5a6546926aff0d04a0280401f939f3a3d8 we have subheadings for these codes anyway, so I think listing all of them here is a bit redundant ... besides this change servers to document the new "PL" selector.